### ISSUE TITLE
How to extract Almanac data from SFRBX message of UBX via RTKLIB.

### ISSUE THREAD

**Issue by zeshanabdullah10** [tomojitakasu/RTKLIB#677]
I want to extract Almanac data from SFRBX message of UBX via RTKLIB. What changes I need to make in the project and in which files/functions.

**Comment by tomojitakasu** [comment]
RTKLIB only supports GPS and QZSS almanac. Other systems are not yet supported.
Read codes of decode_frame_alm() in src/rcvraw.c and decode_rxmsfrbx() in src/rcv/ublox.c.

Otherwise write your own codes by refering ICDs.

**Comment by Waley04** [comment]
Yes, the ICD describes the bit fields necessary for extraction. It is also a matter of referring to the protocol specification, firmware features, and UBX specifications and configuration.
